generic/tests/jumbo.py

```diff
@@ -28,6 +32,31 @@ def run(test, params, env):
     :param params: Dictionary with the test parameters.
     :param env: Dictionary with test environment.
     """
+    def get_ovs_ports(ovs):
+        '''
+        get the ovs bridge all Interface list.
+
+        :param ovs: Ovs bridge name
+        '''
+        cmd = "ovs-vsctl list-ports %s" % ovs
+        return process.getoutput(cmd, shell=True)
```

Comment:
```suggestion
        return host_bridges.list_ports(ovs)
```
When `utils_net.find_bridge_manager(netdst)` returns ovs object, although its name is very long, we can call it normally.
But since we can't get its name accurately, I also agree with your method.